### DISCUSSION TITLE
Se pude crear un grupo en teams?

### DISCUSSION THREAD

**Discussion by StivG18** [Launch-X-Latam/MisionFrontEnd#150] [Q&A]
Me encontré este mensaje de zoe y quiero saber si en verdad si se puede hacer el grupo de teams para comunicarnos entre los explorers.
![image](https://user-images.githubusercontent.com/114199266/194200623-457c7ff1-1c82-4e5f-8afd-3e70863f8ba8.png)

**Comment by doguedogue** [comment]
Excelente, asi mira:

![image](https://user-images.githubusercontent.com/23409026/194383167-9bef0743-58e9-4945-a98f-8100d3b31cc0.png)
![image](https://user-images.githubusercontent.com/23409026/194383290-7fa6f249-59c1-4e87-8039-2293102c8d50.png)

**Comment by VictorFalconi** [comment]
Si se puede? No es todo exclusivo por Github? Confirmen para ver si hago uno :o

**Reply by doguedogue** [reply]
En realidad el grupo ya existe, si quieres comunicar algo está el chat y chats individuales del Teams.

Sin embargo, si el grupo es para algo más especializado, pues si le veo sentido.

Igual que acá en GitHub, el grupo ya existe, la discussion, ya interactuamos. 

Quieres poner un tema, ya están las opciones aquí. Hay para exponer diversos temas y tienes todo el alcance.

Pero no te corto las alas ok, solo te doy mi pov.

**Reply by VictorFalconi** [reply]
Si, entiendo tu punto perfectamente! Pero creo que hay gente que lleva el curso y no tiene tiempo para interactuar o de pronto no le ve sentido a compartir ideas y solo quiere llevar el curso. Es un tanto dificil entablar una conversacion normal cuando se publican tantos temas a la vez, no se puede seguir el ritmo. Si esta permitido, seria interesante contar con la posibilidad de hacerlo

**Reply by Ayuya2001** [reply]
a mi hablar con alguien que es nuevo y compartir ideas, sobre como llevar por ej la practica de abogabot seria muy bueno
